core/src/Lender.sol

```diff
@@ -412,7 +415,6 @@ contract Lender is Ledger {
         emit Transfer(from, address(0), shares);
     }
 
-    /// @dev Note that if `RESERVE` ever gives credit to a courier, its principle won't be tracked properly.
```

Comment:
No longer a concern, because we make sure `RESERVE` _cannot_ credit a courier. Wasn't much of a concern before since `RESERVE`'s abilities are up to us anyway, but it doesn't hurt to be safe here.